components/uuf-renderablecreator-hbs/src/main/java/org/wso2/carbon/uuf/renderablecreator/hbs/helpers/runtime/CssHelper.java

```diff
@@ -48,8 +50,14 @@ public CharSequence apply(String relativePath, Options options) throws IOExcepti
             buffer.append(param);
         }
         buffer.append("\" rel=\"stylesheet\" type=\"text/css\" />\n");
+        String content = buffer.toString();
 
-        addToPlaceholder(buffer.toString(), options);
+        if (isPlacedholderResolved(content, options)) {
+            return "";
+        }
+
+        addToPlaceholder(content, options);
+        ((HashSet<String>) options.data(RESOLVED_PLACEHOLDERS)).add(content);
         return "";
```

Comment:
It seems like this method is always returning a blank string. If so is there any reason to make the return type string/char sequence?